src/main/java/ru/jma/userservice/service/UserService.java

```diff
@@ -0,0 +1,33 @@
+package ru.jma.userservice.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ru.jma.userservice.model.User;
+import ru.jma.userservice.repository.UserRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class UserService {
+
+    @Autowired
+    private UserRepository userRepository;
```

Comment:
Better to inject dependencies through the constructor

Reply:
The way to inject a bean through a constructor is described in the documentation as preferred, but in practice, injecting through a field is more often used